flink-table/flink-table-api-java-bridge/src/test/java/org/apache/flink/table/factories/DataGenTableSourceFactoryTest.java

```diff
@@ -143,6 +144,10 @@ public void testSource() throws Exception {
         descriptor.putLong(DataGenOptions.FIELDS + ".f2." + DataGenOptions.START, 50);
         descriptor.putLong(DataGenOptions.FIELDS + ".f2." + DataGenOptions.END, 60);
 
+        descriptor.putString(
+                DataGenOptions.FIELDS + ".f3." + DataGenOptions.KIND, DataGenOptions.RANDOM);
+        descriptor.putString(DataGenOptions.FIELDS + ".f3." + DataGenOptions.MAX_PAST, "5s");
+
```

Comment:
How about add assertion for the field f3?
Just like what's done in line 156 ~ 159.